blockly-core/core/ui/block_space/block_space.js

```diff
@@ -189,6 +194,12 @@ Blockly.BlockSpace.createReadOnlyBlockSpace = function (container, xml) {
   }, true, true);
 
   var blockSpace = blockSpaceEditor.blockSpace;
+
+  if (opt_options.noScrolling) {
+    blockSpace.scrollbarPair.dispose();
+    blockSpace.scrollbarPair = null;
+  }
```

Comment:
My instinct would be to make it a core feature; it doesn't _sound_ that hard to do. :wink: